main.go

```diff
@@ -35,7 +35,7 @@ func readSettings() []string {
 
 	flag.BoolVar(&s.debug, "debug", false, "print debug output")
 
-	flag.BoolVar(&s.mlock, "mlock", true, "mlock mapped files in memory (only applies to cmd-line specified files).")
+	flag.BoolVar(&s.mlock, "mlock", true, "mlock mapped files in memory rather than copy to heap.")
```

Comment:
on-disk vs in-mem seems like a collection-level attr to me, so it seems like it should be part of the collection spec if anything.

how to get in-mem seems like a server decision, thus the flag.